pyiceberg/table/__init__.py

```diff
@@ -1079,6 +1075,15 @@ def manage_snapshots(self) -> ManageSnapshots:
         """
         return ManageSnapshots(transaction=Transaction(self, autocommit=True))
 
+    def expire_snapshots(self) -> ExpireSnapshots:
+        """
+        Shorthand to run expire snapshots by id or by a timestamp.
+
+        Use table.expire_snapshots().<operation>().commit() to run a specific operation.
+        Use table.expire_snapshots().<operation-one>().<operation-two>().commit() to run multiple operations.
```

Comment:
I believe this isn't true, since the method returns `ExpireSnapshots`, meaning that you cannot chain the operations (which is fine if you call it on the table).
```suggestion
```